hummingbot/strategy_v2/backtesting/executors_simulator/position_executor_simulator.py

```diff
@@ -44,16 +53,25 @@ def simulate(self, df: pd.DataFrame, config: PositionExecutorConfig, trade_cost:
         df_filtered['net_pnl_quote'] = df_filtered['net_pnl_pct'] * df_filtered['filled_amount_quote']
         df_filtered['cum_fees_quote'] = trade_cost * df_filtered['filled_amount_quote']
 
+        # Make sure the stop loss rises with the trailing stop trigger price (if any)
```

Comment:
here the same, is better if we keep them separate as different things. The trailing stop is more like a dynamic take profit, and you can just run the bot without stop loss and with trailing stop